validator/src/admin_rpc_service.rs

```diff
@@ -267,6 +290,29 @@ impl AdminRpc for AdminRpcImpl {
         Ok(())
     }
 
+    fn set_block_engine_config(
+        &self,
+        meta: Self::Metadata,
+        block_engine_url: String,
+        trust_packets: bool,
+    ) -> Result<()> {
+        debug!("set_block_engine_config request received");
+
+        if block_engine_url.contains("http") || block_engine_url.is_empty() {
+            meta.with_post_init(|post_init| {
+                *post_init.block_engine_config.lock().unwrap() = BlockEngineConfig {
+                    block_engine_url,
+                    trust_packets,
+                };
+                Ok(())
+            })
+        } else {
+            Err(jsonrpc_core::error::Error::invalid_params(
+                "block_engine_url must point to an http(s) connection.",
```

Comment:
nit: you can set it to empty string and it works, but that's not a http(s) connection

Reply:
i had fixed this on the backport after testing.  i pulled that commit over to this pr as well

i'm letting the empty string signal disconnect

Reply:
i see, you're referring to the message

already testing this commit, but fixed in next pr